### DISCUSSION TITLE
Is there hosted documentation for older versions of Jolt?

### DISCUSSION THREAD

**Discussion by LPGhatguy** [jrouwe/JoltPhysics#1104] [General]
I'm working on getting Jolt into Rust (<https://github.com/SecondHalfGames/jolt-rust>) and I'd like to link back to the original Jolt documentation for types. In order to prevent my links from bit-rotting as new versions of Jolt are released, is there an existing place where a given version of Jolt's documentation is hosted permanently?

**Comment by LPGhatguy** [comment]
It was a very easy, so I've published the last few major versions of docs as part of [this repo](https://github.com/SecondHalfGames/jolt-docs):
- [5.0.0](https://secondhalfgames.github.io/jolt-docs/5.0.0/)
- [4.0.2](https://secondhalfgames.github.io/jolt-docs/4.0.2/)
- [3.0.1](https://secondhalfgames.github.io/jolt-docs/3.0.1/)

**Comment by jrouwe** [comment]
There is indeed currently no place where older documentation exists. Do you mind if I make a copy of this and host it under my account so that I can keep it up to date in the future?

**Reply by LPGhatguy** [reply]
That sounds great to me! This makes things easier for me, then I can permalink to your copy.

**Comment by jrouwe** [comment]
I created a copy, but for some reason I get an extra 'generated' folder that your version doesn't have.

```
https://jrouwe.github.io/JoltPhysicsDocs/generated/5.0.0/
```

instead of

```
https://jrouwe.github.io/JoltPhysicsDocs/5.0.0/
```

Do you know how to fix that?

**Reply by LPGhatguy** [reply]
Make sure that the "Build and Deployment" section in the Pages settings for the repository is set to "GitHub Actions", like this:

![image](https://github.com/jrouwe/JoltPhysics/assets/654598/87160d35-fb26-4c34-9d18-0eb4ce049e95)

Alternatively, you could probably just flatten the `generated` folder into the root of the repository and delete the GitHub Actions workflow file. The repository doesn't have any build steps, but I went down the GitHub Actions route in case there needed to be some extra processing or other info in the repository in the future.

**Reply by jrouwe** [reply]
Thanks, that worked! The documentation is [available now](https://jrouwe.github.io/JoltPhysicsDocs/).
